examples/collisionavoidance_nonadv.yaml

```diff
@@ -0,0 +1,30 @@
+# Model
+model:
+  time_horizon: 10
+  dt: 1
+
+  vars:
+    state: [x_con x_adv]
+    input: [u]
+    env: [w]
+
+  bounds:
+    x_con: (-10, 10)
+    x_adv: (-10, 10)
+    u: (0.99, 1.01)
+
+# Specification
+    
+init:
+  - stl: x_con = -5
+  - stl: x_adv = -5
+
+sys:
+  - name: NonAdv_AvoidCollision
+    stl: "G[0,10](w == 1)"
```

Comment:
There can only be one stl per list item (bulletin -)
You don't have to name all of them if that's a pain, the name is just extra metadata I thought might be useful later.